CHANGELOG.md

```diff
@@ -4,6 +4,60 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.6.0
+
+### Important Changes
+
+- **feat(metrics): Add `timings` method to metrics (#12226)**
+
+  This introduces a new method, `metrics.timing()`, which can be used in two ways:
+
+  1. With a numeric value, to simplify creating a distribution metric. This will default to `second` as unit:
+
+  ```js
+  Sentry.metrics.timing('myMetric', 100);
+  ```
+
+  2. With a callback, which will wrap the duration of the callback. This can accept a sync or async callback. It will
+     create an inactive span around the callback and at the end emit a metric with the duration of the span in seconds:
+
+  ```js
+  const returnValue = Sentry.metrics.timing('myMetric', measureThisFunction);
+  ```
+
+- **feat(react): Add `Sentry.reactErrorHandler` (#12147)**
+
+  This PR introduces `Sentry.reactErrorHandler`, which you can use as follows:
```

Comment:
```suggestion
  This PR introduces `Sentry.reactErrorHandler`, which you can use in React 19 as follows:
```

I think this is react 19+ only?

Reply:
Good point, I think this is react 19+ only.